tools/not.cpp

```diff
@@ -24,9 +24,12 @@
 #include <windows.h>
 #endif
 
-#ifdef __APPLE__
+#if defined(__unix__) || defined(__APPLE__)
 #include <spawn.h>
 #include <sys/wait.h>
+#endif
+
+#ifdef __APPLE__
 #include <TargetConditionals.h>
```

Comment:
`TargetConditionals.h` is only needed for `TARGET_OS_IPHONE`, so I think we can drop this.

Reply:
Should the `#ifdef` be changed to `TARGET_OS_IPHONE` then?